### PR TITLE
create bucket if not exist in script

### PR DESCRIPTION
**Pull request by agathanatasha**
Fix get started model script in readme
- creates bucket with default bucket name if one does not exist

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -125,11 +135,7 @@ if __name__ == "__main__":
 
 Run both cells and observe that your model metrics are recorded in MLflow!
 
-To access MLflow, run:
-```
-microk8s kubectl get services -A|grep "mlflow-server"
-```
-And open the `mlflow` `ClusterIP` in the browser with `:5000` on the end.
+To access MLflow dashboard, go to [http://10.64.140.43.nip.io/mlflow/#/](http://10.64.140.43.nip.io/mlflow/#/)
```
[the remark this answers is not in the document]
Comment (agathanatasha) [reply README.md:138]: The instructions tell people to deploy the whole kubeflow. All of the setup would be included in that

**Comment by agathanatasha** [comment]
merging PRs

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ca-scribner) [reply README.md:131]: ok that's fair.  Maybe if we just make a comment saying "connect to the ingress_ip/mlflow, or if you followed the above instructions `10.64...`?

[the remark this answers is not in the document]
Comment (agathanatasha) [reply README.md:131]: updated in pr #29

**Pull request merged**